app/assets/javascripts/tests/formatSessionForList.test.js

```diff
@@ -0,0 +1,63 @@
+import test from 'blue-tape';
```

Comment:
Any reasons for calling the test file differently than the code file. I liked the structure we had so far, it's intuitive to find the tests there.

Reply:
We can improve that. Reason is that this has 0 to do with angular. But I'm open to proposals